OpenRA.Game/WRot.cs

```diff
@@ -29,6 +29,27 @@ public struct WRot : IEquatable<WRot>
 		public static WRot operator -(WRot a, WRot b) { return new WRot(a.Roll - b.Roll, a.Pitch - b.Pitch, a.Yaw - b.Yaw); }
```

Comment:
The + and - operators should really be removed, since they are not valid operators for 3d-rotations. They only accidentally give an answer that seems to make sense when pitch and roll are zero.

Reply:
Good idea, and their use is exactly why things break so badly when introducing slopes.

I'll add this to my list of things to do in the next PR, which removes their from turrets etc.